docs/Table.md

```diff
@@ -72,86 +99,192 @@ const { query } = initialize({ iotsitewiseClient });
 <iot-table
   viewport={{ duration: '5m' }}
   
-  tableColumns={[
+  items={[
     {
-      header: 'Tempature',
-      rows: [
-        toId({ assetId: 'engine-1', propertyId: 'tempature' }),
-        toId({ assetId: 'engine-2', propertyId: 'tempature' }),
-      ]
+      rpm: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-1', propertyId: 'rotation_per_minute' }),
+          resolution: 0
+        }
+      },
+      wind_speed: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-1', propertyId: 'wind_speed' }),
+          resolution: 0
+        }
+      }
     },
     {
-      header: 'RPM',
-      rows: [
-        toId({ assetId: 'engine-1', propertyId: 'rpm' }),
-        toId({ assetId: 'engine-2', propertyId: 'rpm' }),
-      ]
+      rpm: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-2', propertyId: 'rotation_per_minute' }),
+          resolution: 0
+        }
+      },
+      wind_speed: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-2', propertyId: 'wind_speed' }),
+          resolution: 0
+        }
+      }
+    }
+  ]}
+  
+  columnDefinitions={[
+    {
+      key: 'rpm',
+      header: 'RPM'
+    },
+    {
+      key: 'wind_speed',
+      header: 'Wind Speed'
     }
   ]}
   
   queries={[
     query.timeSeriesData({
       assets: [
         { 
-          assetId: 'engine-1', 
+          assetId: 'turbine-1', 
           properties: [
-            { propertyId: 'tempature' },
-            { propertyId: 'rpm' }
+            { propertyId: 'rotation_per_minute' },
+            { propertyId: 'wind_speed' }
           ],
-        }, { 
-          assetId: 'engine-2', 
+        }, 
+        { 
+          assetId: 'turbine-2', 
           properties: [
-            { propertyId: 'tempature' },
-            { propertyId: 'rpm' }
-          ],
+            { propertyId: 'rotation_per_minute' },
+            { propertyId: 'wind_speed' }
+          ]
         }
-    ])
-  ]}}
+      ]
+    })
+  ]}
 />
 ```
 
 ## Properties
 
-The table component contains the following properties that you can customize. 
+The table component contains the following customizable properties:
 
-### `tableColumns`
+Note: Since the table component built over the `@awsui/components-react` table component, please refer to [AWS-UI/table](https://github.com/aws/awsui-documentation/blob/main/components/table.md) about most of the available properties. Differences between AWSUI/Table and this Table component are listed below.
 
-Specifies the row and column structure for the table, provides a mechanism to map the time series data queried for, into the cells of your choice
-Type: Array
+### `items`
+Specifies the data displayed in rows. Each item contains the data for one row. The display of a row is handled by the cell property of each column definition in the columnDefinitions property.
 
-Each table column contains the following properties
+Type: Array of [Item](#item) objects
 
-`header`
+Required: Yes
```

Comment:
The style is to only specify if optional, by stating `(Optional)` at the start of the paragraph. No need to state if required